package.json

```diff
@@ -44,7 +44,7 @@
     "lodash": "^3.10.0",
     "method-override": "^2.3.3",
     "mocha": "~1.20.0",
-    "mongoose": "^4.0.6",
+    "mongoose": "4.1.11",
```

Comment:
It's definitely an authentication issue. I think we should require the `user` and `pass` options to be passed along to the `connect` method.

After reviewing it further last night, I came to the conclusion that this isn't a bug. It appears to be a limitation of how we're handling authentication.

I'm don't have any issues with Mongoose `4.1.12` locally when running with the Development env config, because I use local.js to manage the `db` setting; I'm setting `user` and `pass` there. However, I did run into an issue with running in Test env. Because the `db` setting is not using `user` and `pass`. 

Thus, why I added #1011